apphome/stats.go

```diff
@@ -70,3 +71,50 @@ func messagesAnalyzed(userIdFilt expression.ConditionBuilder) int {
 	}
 	return val
 }
+
+func userProj()	expression.ProjectionBuilder {
+	return expression.NamesList(expression.Name("user_id"))
+}
+
+func UsersWithBadQualityMessages() ([]string, error) {
```

Comment:
```suggestion
func UsersIdsWithBadQualityMessages() ([]string, error) {
```
I know it sounds weird, but then it makes it more obvious that the return is a []string of userIds and not userNames

Reply:
Also, this is meant only for use with notifications. Shouldn't you move this to the notification package ?

Reply:
Removed